src/resolvers/get-make.ts

```diff
@@ -758,6 +758,8 @@ function getMakeFromDescription(description: string): Make | null {
     return Make.SKODA
   } else if (description.match(/(volkswagen|vw)/i)) {
     return Make.VOLKSWAGEN
+  } else if (description.match(/ligier/i)) {
+    return Make.LIGIER
```

Comment:
`VJR` should be mapped to Ligier as well: https://en.wikibooks.org/wiki/Vehicle_Identification_Numbers_(VIN_codes)/World_Manufacturer_Identifier_(WMI)

Reply:
Whoops. I didn't open the correct link so I couldn't find the 3-letter code for ligier. Adding.